docs/pages/server-access/guides/openssh.mdx

```diff
@@ -42,7 +71,7 @@ On your local machine, print the Teleport certificate authority certificate to
 stdout:
 
 ```code
-$ tctl auth export --type=user | sed "s/cert-authority\ //"
+$ tctl auth export --type=openssh | sed "s/cert-authority\ //"
```

Comment:
FYI, @marcoandredinis made it possible to curl this info without requiring you to be logged in. It's a bit more convenient because you can run curl directly from the openssh node rather than transferring a file to the node.

Reply:
Looks like @marcoandredinis also opened a PR to update the same doc page: https://github.com/gravitational/teleport/pull/23560 kinda a chicken and egg problem...

Reply:
Feel free to keep it as it
I'll update it after you merge your PR :+1:

Reply:
Sweet thank you!